docs/product/components/modals.html

```diff
@@ -57,30 +148,32 @@ <h1 class="s-modal--header" id="modal-title">Example danger title</h1>
     </div>
     <div class="stacks-preview">
 {% highlight html %}
-<aside class="s-modal js-modal-overlay js-modal-close" id="modal-base" tabindex="-1" role="dialog" aria-labelledby="modal-title" aria-describedby="modal-description" aria-hidden="true">
+<aside class="s-modal js-modal-overlay js-modal-close" id="modal-base" tabindex="-1" role="dialog" aria-labelledby="modal-title" aria-describedby="modal-description" aria-hidden="true"
+    data-controller="s-modal" data-target="s-modal.modal">
     <div class="s-modal--dialog js-modal-dialog" role="document">
         <h1 class="s-modal--header" id="modal-title">…</h1>
         <p class="s-modal--body" id="modal-description">…</p>
         <div class="grid gs8 gsx s-modal--footer">
             <button class="grid--cell s-btn s-btn__primary js-modal-primary-btn" type="button">…</button>
-            <button class="grid--cell s-btn js-modal-cancel-btn" type="button">…</button>
+            <button class="grid--cell s-btn js-modal-cancel-btn" type="button" data-action="s-modal#toggle">…</button>
         </div>
-        <a class="s-modal--close s-btn s-btn__muted js-modal-close" href="#" aria-label="Close">
+        <a class="s-modal--close s-btn s-btn__muted js-modal-close" href="#" aria-label="Close" data-action="s-modal#toggle">
```

Comment:
Not specifically related to this PR, but it would be great if these were `<button type="button">` elements.